app/templates/macros/page_macros.html

```diff
@@ -7,24 +7,19 @@
        <button class="ui button start-edit">
         Edit
        </button>
-       <p class= "updated_alert">Updated successfully</p>
-       <button class="ui primary button end-edit">
+       <button class="ui primary button end-edit" style="display:none">
```

Comment:
With brandon's new merges into master we can take out the `style="display:none"` on the Save, Cancel and Update message since they are specified now in scss - you'll also have rename the `updated_alert` to `updated-alert` in the respective scss file - i think is `admin.scss` - like you did for the JS stuff